stdlib/2/typing.pyi

```diff
@@ -2,6 +2,7 @@
 
 from abc import abstractmethod, ABCMeta
 from types import CodeType, FrameType, TracebackType
+import collections  # Needed by aliases like DefaultDict, Deque, etc.
```

Comment:
Could you add a reference to the mypy issue that makes clear why mypy needs this import? It's not really obvious from this comment why the import is necessary.